crates/core/src/delta_datafusion/cdf/scan.rs

```diff
@@ -0,0 +1,93 @@
+use std::any::Any;
+use std::fmt::Formatter;
+use std::sync::Arc;
+
+use arrow_schema::{DataType, Field, SchemaRef, TimeUnit};
+use datafusion::execution::{SendableRecordBatchStream, TaskContext};
+use datafusion::physical_plan::{DisplayAs, DisplayFormatType, ExecutionPlan};
+use datafusion_physical_expr::{Partitioning, PhysicalSortExpr};
+use lazy_static::lazy_static;
+
+/// Change type column name
+pub const CHANGE_TYPE_COL: &str = "_change_type";
+/// Commit version column name
+pub const COMMIT_VERSION_COL: &str = "_commit_version";
+/// Commit Timestamp column name
+pub const COMMIT_TIMESTAMP_COL: &str = "_commit_timestamp";
+
+lazy_static! {
+    pub static ref CDC_PARTITION_SCHEMA: Vec<Field> = vec![
+        Field::new(COMMIT_VERSION_COL, DataType::Int64, true),
+        Field::new(
+            COMMIT_TIMESTAMP_COL,
+            DataType::Timestamp(TimeUnit::Millisecond, None),
```

Comment:
Should be UTC since it's a delta timestamp

Reply:
Oddly Spark does not write the CDF timestamp with any timezone, so this doesn't actually work

Reply:
Isn't that then a bug. They mention timestamp in the docs, and protocol states a timestamp is an UTC datetime